modules/dnn/src/tflite/tflite_importer.cpp

```diff
@@ -25,7 +25,7 @@ using namespace opencv_tflite;
 // This values are used to indicate layer output's data layout where it's possible.
 // Approach is similar to TensorFlow importer but TFLite models do not have explicit
 // layout field "data_format". So we consider that all 4D inputs are in NHWC data layout.
-enum DataLayout
+enum TFDataLayout
```

Comment:
let's reuse DataLayout

Reply:
Got it.

Reply:
Hi @vpisarev, the `DataLayout` has been reused.